modules/python/test/test_unicode_support.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding:utf-8 -*-
+from __future__ import print_function
+
+import numpy as np
+import cv2 as cv
+import os
+import tempfile
+import sys
+
+from tests_common import NewOpenCVTests
+
+str_unicode = u"测试tést"
+
+
+def rmdir(directory):
```

Comment:
Use [`shutil.rmtree`](https://docs.python.org/3/library/shutil.html#shutil.rmtree) instead of manual `os.walk`